doc/Makefile

```diff
@@ -1,195 +0,0 @@
-# Makefile for Sphinx documentation
```

Comment:
We still need to build sphinx docs in some though? `doc/` is more standard for this than `docs/` (see e.g. numpy, scipy pandas etc)

Reply:
Ok, I can revert that. Was just to bring it in Line with the other packages in the GeoStat Framework.

Reply:
I will keep it that way (docs). So settings on RTD are consistent. Hope you don't mind.

Reply:
Ok, it's not too critical either way.